kv/store.go

```diff
@@ -142,12 +169,22 @@ func (store Store) createJobsOps(conn *connection.Connection) ([]etcd.Op, error)
 			return []etcd.Op{}, err
 		}
 
-		ops = append(ops, etcd.OpPut(fmt.Sprintf("%s/%s%s", conn.ID, jobsDir, job.ID), string(value)))
+		ops = append(ops, etcd.OpPut(fmt.Sprintf("%s%s/%s%s", connectionsPrefix, conn.ID, jobsDir, job.ID), string(value)))
 	}
 
 	return ops, nil
 }
 
+// GetPrefix returns the default prefix for the kv set
+func GetPrefix() string {
+	return prefix
```

Comment:
What's are the benefits of having public functions vs. public consts?

Reply:
I'm partial to `getters` and `setters` logic more than global variables. Your call, what would you prefer?